internal/cli/internal/command/services/postgres/connect.go

```diff
@@ -0,0 +1,93 @@
+package postgres
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	"github.com/superfly/flyctl/internal/cli/internal/command"
+	"github.com/superfly/flyctl/internal/cli/internal/command/ssh"
+
+	"github.com/superfly/flyctl/internal/cli/internal/flag"
+	"github.com/superfly/flyctl/internal/client"
+	"github.com/superfly/flyctl/pkg/agent"
+)
+
+func newConnect() (cmd *cobra.Command) {
+	const (
+		long = `
+			Connect to the Postgres console
```

Comment:
this should't be tabbed; it'll be indented when printed to the output. [Here](https://github.com/superfly/flyctl/blob/79ff1a15d3d3eef7b15b27a084fb84ad6679cbe0/internal/cli/internal/command/apps/apps.go#L13) is a correct example